scripts/filter_multimappers.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+"""Filter multimappers in a SAM file by indel count.
+
+This script uses the pysam library to open the input SAM file and iterates
+over each alignment in it. It is assumed that the file is ordered by query
+name. For each alignment, it checks if it is a secondary or supplementary
+alignment, if it is, it skips to the next alignment. Otherwise, it adds the
+alignment to a list storing all alignments with the same query name.
+If the current query changes, it selects the best alignment(s) from the list
+of current alingments and writes them to the standard output. Current query
+and current alignments list are reset to the new query and alignment
+respectively.
+Finally, if there are any remaining alignments in the current alignments
+list and selects the best alignment(s) for those queries, writing them to
+standard output.
+
+Input:
+    input_sam_file (str): Path to the input SAM file, ordered by query name.
+
+Output:
+    standard output: Filtered SAM file
+
+Functions:
+    parse_arguments():
+        Command-line arguments parser
+    count_indels(pysam.AlignedSegment):
+        Count the number of indels in an alignment based on its CIGAR string.
+    find_best_alignments(list[pysam.AlignedSegment]):
+        Find alignments with less indels
+    write_output(list[pysam.AlignedSegment]):
+        Write the output to the standard output (stdout).
+    main(Path):
+        Filter multimappers by indels count.
+
+Usage:
+    python filter_multimappers.py -i input_sam_file
+    python filter_multimappers.py -i input_sam_file > output_sam_file
```

Comment:
Given that you put a shebang in the first line, if you make the script executable, you don't need to include `python` here. Also, use the metavar like `FILE` or `PATH` or `SAM` instead of `input_sam_file`:

```bash
filter_multimappers.py SAM
```